### ISSUE TITLE
Download not supported

### ISSUE THREAD

**Issue by hjndkj** [fluttercommunity/flutter_webview_plugin#591]
Clicking URL resource cannot be downloaded directly

**Comment by charafau** [comment]
I think this is out of scope for now - ideally we would like to move whole project to use official flutter webview plugin when it's stable

**Comment by brunozanoelo** [comment]
Nothing has solved so far. It seems they are focusing in this situation right now:
"We are working closely with the Flutter Team to integrate all the Community Plugin features in the Official WebView Plugin. We will try our best to resolve PRs and Bugfixes, but our priority right now is to merge our two code-bases. Once the merge is complete we will deprecate the Community Plugin in favor of the Official one."

I´ve spent hours in this situation and tried many plugins, nothing solved.
